jsx/FilterableDataTable.js

```diff
@@ -1,10 +1,11 @@
-import React, {Component} from 'react';
+import React, {PureComponent} from 'react';
```

Comment:
not in **24.1**

_although purecomponents seem interesting from a performance stand-point, I'm not sure if this is necessary here and can likely be removed so as to not cause trouble_

to be tested, although basic assumption is to adopt **24.1**